ajax/libs/slim.js/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "slim.js",
+  "filename": "slim.js",
```

Comment:
Can you change `filename` to `slim.min.js`?
We want to provide users minified versions too.
Are you convenient to generate minified versions and upload here?
If not, we can do it for you.